pkg/kubelet/kubelet.go

```diff
@@ -460,6 +468,42 @@ func NewMainKubelet(
 			return nil, err
 		}
 		klet.containerRuntime = rktRuntime
+	case "remote":
```

Comment:
Do we assume `remote` runtime will be used for any runtime server like frakti/oci-runtime/xxx ?